lib/xdrgen/generators/rust/src/types.rs

```diff
@@ -154,9 +159,6 @@ impl From<Error> for () {
     fn from(_: Error) {}
 }
 
-#[allow(dead_code)]
-type Result<T> = core::result::Result<T, Error>;
```

Comment:
The type alias is removed because the serde_as crate assumes `Result` in the current namespace is the `core::result::Result`.